west.yml

```
@@ -56,7 +56,7 @@ manifest:
       revision: f1fa8241f8786198ba41155413243de36ed878a5
       path: modules/hal/infineon
     - name: hal_nordic
-      revision: 7430036d4f18cdf1f49158e97e5d57aacb44fb92
+      revision: pull/61/head
```

@mniestroj Please update to:
```suggestion
      revision: 031065e8f9066b4e5fcda57f390af1cd67f35a65
```
as https://github.com/zephyrproject-rtos/hal_nordic/pull/61 got merged.